tools/bootnode/bootnode.go

```diff
@@ -180,13 +184,25 @@ func createLocalNode(privKey *ecdsa.PrivateKey, ipAddr net.IP, port int) (*enode
 		return nil, errors.Wrap(err, "Could not open node's peer database")
 	}
 
+	forkID := &pb.ENRForkID{
+		CurrentForkDigest: []byte{0, 0, 0, 0},
```

Comment:
Is it okay to make this hardcoded?

Reply:
So in the context of bootnodes, we have no way to know what the current fork digest is of a chain. Because you need both genesis time + validator root. Therefor we just make it a zeroed out byte array for now. Bootnodes are a special case here, so we just ignore the fact that they have different fork digests for now in the beacon node.

Reply:
Great! Got it!